components/ParameterForm.vue

```diff
@@ -142,6 +142,10 @@ import type { Parameter, ParameterSet, ValueData } from "@/types/parameterTypes"
 import { TypeOfParameter } from "@/types/parameterTypes";
 import type { NewScenarioData } from "@/types/apiResponseTypes";
 
+const props = defineProps<{
+  inModal?: boolean
+}>();
```

Comment:
I think it would be fine to make this required, and include isModal=false for the instance in new scenario page.